swagger/v1-prerelease/swagger.json

```diff
@@ -0,0 +1,3455 @@
+{
```

Comment:
At some point we should invest in better examples in the document for our data models